src/thyra/ThyraAccount.ts

```diff
@@ -140,4 +142,41 @@ export class ThyraAccount implements IAccount {
     }
     return signOpResponse.result;
   }
+
+  /**
+   * This method aims to buy rolls on behalf of the sender.
+   *
+   * @param amount - The amount of rolls to be bought.
+   * @param fee - The fee to be paid for the transaction.
+   * @returns An ITransactionDetails object. It contains the operationId on the network.
+   */
+  buyRolls(amount: string, fee: string): Promise<ITransactionDetails> {
+    throw new Error('Method not implemented.');
```

Comment:
Can you create an issue on Thyra side if this is not yet done ?

Reply:
Sure, I will ,but need to ask the inno team about this